src/parser/config/nodeTypesConfig.json

```diff
@@ -1518,8 +1518,8 @@
     },
     {
         "expression": "class",
-        "metrics": [],
-        "type": "keyword",
+        "metrics": ["classes"],
+        "type": "statement",
```

Comment:
Info: counts now also for classes defined via class expressions and not only class declarations.